src/app/@ansyn/core/components/imagery-status/imagery-status.component.ts

```diff
@@ -65,6 +66,13 @@ export class ImageryStatusComponent implements OnInit, OnDestroy {
 			distinctUntilChanged()
 		);
 
+	copyOriginalOverlayDataFlag$: Observable<boolean> = this.core$
+		.pipe(
+			pluck<ICoreState, boolean>('enableDownloadOriginalOverlayData'),
```

Comment:
Don't use pluck() or distinctUntilChanged(). Instead, add a selector for enableDownloadOriginalOverlayData at the bottom of core.reducer.ts. Use that selector instead of this.core$